tests/data/aws/ec2.py

```diff
@@ -15,6 +15,41 @@
     ],
 }
 
+DESCRIBE_TAGS = {
```

Comment:
Thanks very much for including mock data and integration tests.  Can you also please add an entry to docs/schema.md?